ISSUE_TEMPLATE/01_bug_report.md

```diff
@@ -0,0 +1,24 @@
+---
+name: ":bug: Bug report"
+about: Tell us about a problem you are experiencing
+---
+
+<!--
+1. Describe the issue you are having and what you expected to happen.
+-->
+
+<!--
+2. Describe the steps to reproduce, including any log snippets that will help diagnose.
+-->
+
+<!--
+3. Describe your environment, including
+   - Version
+   - Deployment environment (e.g. VMware appliance, EC2 appliance, Containers, etc)
```

Comment:
This part was interesting.  Since this is at the org level, this template will apply to everything (the ManageIQ appliancation, gems, build repos, docs, etc).  I wasn't sure how to present this for all 3, so I'm up for some help here.  The goal is short and sweet, so I don't want to put too much to be annoying.

Reply:
Is there a magic variable for current repo that we could put context specific information?  For gems like kubeclient, optimist, linux_admin, etc. this template could be unexpected

Reply:
No it's just a straight text file.  It we want to override we have to create a file in the repo itself

Reply:
Maybe a few of us can jump on a call, merge this, and see how it looks for several repos and add any overrides for repos where this template would be confusing, such as: kubeclient, optimist, linux_admin.